lib/glue/tasks/bundle-audit.rb

```diff
@@ -44,6 +44,14 @@ def supported?
     end
   end
 
+  private
+  def normalize_unknown_severity sev
+    sev = '' if sev.nil?
+    return sev if sev.strip.chomp.downcase != "unknown"
+    Glue.warn "Criticality: Unknown. Setting it to medium"
```

Comment:
Maybe this should be info?

Reply:
Do you mean the logging class or the severity type?

Reply:
Severity, I've merged either way - as it's not critical. But I think this can be  info.

Reply:
OK, fair enough. The point of being medium was to be a more conservative approach since info issues tend to be more ignored. On the other hand, this approach leads to more false positives.
If you think it makes more sense to have the info, I can make a new PR to change that.

Reply:
I don't have much insight about it because I've never used this task. My thought was - a warning is something the user should respond to, as Glue might not behave well. I wasn't sure this is the case...